packages/web/src/store/tipping/sagas.ts

```diff
@@ -209,6 +212,50 @@ function* refreshSupportAsync({
   }
 }
 
+function* fetchSupportingForUserAsync({
+  payload: { userId }
+}: {
+  payload: { userId: ID }
+  type: string
+}) {
+  const encodedUserId = encodeHashId(userId)
+  if (encodedUserId) {
```

Comment:
stylistic nit: we're nesting the happypath here, we could just return early if `!encodedUserId`